conf/nd96asr_v4.conf

```diff
@@ -8,10 +8,39 @@
 # Fri Jan 14 21:17:09 UTC 2022
 #
 
+#######################################################################
```

Comment:
do you think we can unify the conf files for NDv4 40 and 80 GB SKU's?

Reply:
The reason they are separate is due to differences in the expected values of "check_hw_physmem" tests. Ndv4 80G has more main memory.

The tests:
80G sku: * || check_hw_physmem 1857830712kB 1857830712kB 3%
40G sku:  * || check_hw_physmem 928858388kB 928858388kB 3%

aside from this single test the files are pretty much the same.

The options are:
- Keep individual files
- Unify them and instruct the user to modify the single test based on sku